sm-core-model/src/main/java/com/salesmanager/core/model/catalog/category/image/CategoryImage.java

```diff
@@ -0,0 +1,84 @@
+package com.salesmanager.core.model.catalog.category.image;
+
+import com.salesmanager.core.constants.SchemaConstant;
+import com.salesmanager.core.model.catalog.category.Category;
+import com.salesmanager.core.model.generic.SalesManagerEntity;
+
+import javax.persistence.*;
+import java.io.InputStream;
+import java.util.UUID;
+
+/**
+ * @author Mostafa Saied <mostafa.saied.fci@gmail.com>
+ */
+
+@Entity
+@Table(name = "CATEGORY_IMAGE", schema= SchemaConstant.SALESMANAGER_SCHEMA)
+public class CategoryImage extends SalesManagerEntity<Long, CategoryImage> {
+
+    @Id
+    @Column(name = "CATEGORY_IMAGE_ID")
+    @TableGenerator(name = "TABLE_GEN", table = "SM_SEQUENCER", pkColumnName = "SEQ_NAME", valueColumnName = "SEQ_COUNT", pkColumnValue = "PRODUCT_IMG_SEQ_NEXT_VAL")
+    @GeneratedValue(strategy = GenerationType.TABLE, generator = "TABLE_GEN")
+    private Long id;
+
+
+    @ManyToOne
+    @JoinColumn(name = "CATEGORY_ID", nullable = false)
+    private Category category;
+
+    @Column(name = "CATEGORY_IMAGE")
+    private String categoryImage;
+
+    @Column(name = "DEFAULT_IMAGE")
+    private boolean defaultImage = false;
+
+    @Transient
+    private InputStream image = null;
+
+    public CategoryImage(){
+    }
+
+    public String getCategoryImage() {
```

Comment:
why not to introduce lombok here?

Reply:
Again, The whole project not using Lombok, that's why I didn't use it.